generate.go

```diff
@@ -107,6 +107,8 @@ BuildRR:
 					mod, offset, err = modToPrintf(s[j+2 : j+2+sep])
 					if err != nil {
 						return err.Error()
+					} else if (start + offset) < 0 || (end + offset) > 1<<32-1 {
```

Comment:
These parenthesis aren’t needed, can they be removed?

Reply:
Certainly.

Reply:
done + added a test to keep codecov happy